source-api/src/commonMain/kotlin/eu/kanade/tachiyomi/source/online/HttpSource.kt

```
@@ -404,3 +404,5 @@ abstract class HttpSource : CatalogueSource {
      */
     override fun getFilterList() = FilterList()
 }
+
+class LicensedMangaChaptersException : Exception("Licensed - No chapters to show")
```

Shouldn't this use the newly added `licensed_manga_chapters_error` from `strings.xml`

---

Resolution into a string needs a context which is not available here. This string is left as a fallback so the message will appear in logs and in case exception is not handled by formattedMessage but shown to a user, there's still some text.